brainstormforce/ultimate-addons-for-gutenberg
# Fix: Info-box - Button link setting “Open in New Window” is placed under the Link panel.

### Description

- The info-box button link setting should have “Open in New Window” just under the Link panel. Currently it is on the last order.

### Screenshots

- https://share.getcloudapp.com/bLulKgl0

### Types of changes




- Bug fix (non-breaking change which fixes an issue)

### How has this been tested?

- Drag and drop info-box.
- Go to Call To Action panel.
- Click Type to button, text or complete box.
- Make sure “Open in New Window” should be placed below the Link option.

### Checklist:
